<launch>
  <arg name="robot_id" value="$(optenv ROBOT_ID unnamed_robot)"/>

  <group ns="$(arg robot_id)">
    <node pkg="joy" type="joy_node" name="driver_joy" output="screen">
      <param name="autorepeat_rate" value="10"/>
      <param name="coalesce_interval" value="0.05"/>
    </node>

    <node pkg="sdpo_driver_omnijoy" type="sdpo_driver_omnijoy_node"
          name="sdpo_driver_omnijoy" output="screen">
      <remap from="~/cmd_vel" to="cmd_vel"/>
      <param name="axis_linear_x" value="1"/>
      <param name="axis_linear_y" value="0"/>
      <param name="axis_angular" value="2"/>
      <param name="axis_deadman" value="4"/>
      <param name="axis_turbo" value="5"/>
      <param name="axis_turbo_up" value="7"/>
      <param name="axis_turbo_down" value="6"/>
      <param name="scale_angular" value="1.0"/>
      <param name="scale_linear" value="0.4"/>
      <param name="turbo_scale_linear" value="0.25"/>
      <param name="turbo_max_scale_linear" value="0.4"/>
      <param name="turbo_scale_angular" value="0.40"/>
    </node>
  </group>
</launch>
